docs/en/mkdocs/search_index.json

```diff
@@ -72,7 +72,7 @@
    automationName: \nUiAutomator2\n\n  }\n};\n\nasync function main () {\n  const client = await wdio.remote(opts);\n\n  await client.deleteSession();\n}\n\nmain();\n\n\n\n\nRunning Test Commands\n\n\nYou can see that we've specified our Appium port and also constructed our\nDesired Capabilities to match our requirements (but don't forget to replace the\npath with the actual download path for your system). We've registered this fact\nwith \nwebdriverio\n and now have a client object which will represent the\nconnection to the Appium server. From here, we can go ahead and start the\nsession, perform some test commands, and end the session. In our case, we will\nsimply type into a text field and check that the correct text was entered:\n\n\n// javascript\n\nconst field = await client.$(\nandroid.widget.EditText\n);\nawait field.setValue(\nHello World!\n);\nconst value = await field.getText();\nassert.equal(value, \nHello World!\n);\n\n\n\n\nWhat's going on here is that after creating a session and launching our app,\nwe're instructing Appium to find an element in the app hierarchy and type into\nit. The same field is then queried for its text, which is asserted to be what we\nexpect.\n\n\nPutting it all together, the file should look like:\n\n\n// javascript\n\nconst wdio = require(\nwebdriverio\n);\nconst assert = require(\nassert\n);\n\nconst opts = {\n  path: '/wd/hub',\n  port: 4723,\n  capabilities: {\n    platformName: \nAndroid\n,\n    platformVersion: \n8\n,\n    deviceName: \nAndroid Emulator\n,\n    app: \n/path/to/the/downloaded/ApiDemos.apk\n,\n    appPackage: \nio.appium.android.apis\n,\n    appActivity: \n.view.TextFields\n,\n    automationName: \nUiAutomator2\n\n  }\n};\n\nasync function main () {\n  const client = await wdio.remote(opts);\n\n  const field = await client.$(\nandroid.widget.EditText\n);\n  await field.setValue(\nHello World!\n);\n  const value = await field.getText();\n  assert.equal(value,\nHello World!\n);\n\n  await client.deleteSession();\n}\n\nmain();\n\n\n\n\nYou can try and run this test on your own. Simply save it and execute it using\n\nnode\n:\n\n\nnode index.js\n\n\n\n\nIf everything is set up correctly, you'll see Appium begin spitting out\nlots of logs and eventually the app will pop up on the screen and start\nbehaving as if an invisible user were tapping on it!\n\n\nWhat's Next\n\n\nWe've only scratched the surface of what you can do with Appium. Check out\nthese resources to help you on your journey:\n\n\n\n\nThe Appium \ncommand reference\n - learn about what commands are available, how to use them with specific client libraries, etc...\n\n\n\n\nThe \nsample-code\n directory, where lots more code samples are available\n\n\n\n\n\n\ndiscuss.appium.io\n - this is the Appium community forum, which is a great first place to go for help getting started, or if you think you may have run into a bug\n\n\n\n\nThe Appium \nissue tracker\n - let the Appium maintainers know here if you think you've found a bug", 
```

Comment:
is this file autogenerated?

Reply:
Yes. Maybe `path: '/wd/hub',` in getting started was added.